stackit/internal/services/iaas/publicip/resource_test.go

```diff
@@ -179,6 +187,7 @@ func TestToUpdatePayload(t *testing.T) {
 				Labels: &map[string]interface{}{
```

Comment:
you should add a test case here where the field is not set in the Model and check that in the payload it is sent as  nullable string with value nil

Reply:
this test case is already in the default_ok. But I will align it with the CreatePayload test. Default test will have some values and I will create a new test for network_interface_nil

Reply:
Extended unit tests